.github/CODEOWNERS

```diff
@@ -0,0 +1,7 @@
+# CODEOWNERS: https://help.github.com/articles/about-codeowners/
+
+# Primary repo maintainers
+*       @ilgooz @jeronimoalbi @Pantani
```

Comment:
GitHub is warning that `@Pantani` is an unknown owner. Maybe a permissions issue?

Reply:
yes, weird, maybe because I opened the PR from a fork. I think it will solve after the merge